src/web/app/components/question-types/question-statistics/rubric-question-statistics.component.html

```diff
@@ -8,33 +8,15 @@
     </div>
     <div class="col-sm-8" style="text-align: right;">
       <label class="form-check-label">
-        <input type="checkbox" [(ngModel)]="excludeSelf" class="form-check-input"> <span ngbTooltip="Excludes giver's responses to himself/herself from statistics">Exclude self evaluation</span>
+        <input type="checkbox" [(ngModel)]="excludeSelf" (ngModelChange)="getTableData()" class="form-check-input"> <span ngbTooltip="Excludes giver's responses to himself/herself from statistics">Exclude self evaluation</span>
```

Comment:
Adding `(ngModelChange)="getTableData()` here because `ngOnChanges` does not get called when the checkbox is toggled.